### ISSUE TITLE
[use-baseline] For unavailable attribute value warnings, set the character position to the start of value

### ISSUE THREAD

**Issue by rviscomi** [yeonjuan/html-eslint#313]
Demo: https://github.com/rviscomi/baseline-demos/tree/main/eslint#html-eslint

Sample HTML:

```html
<a href="#old" referrerpolicy="origin-when-cross-origin">Old Stuff</a>
```

`eslint.config.mjs`:

```js
  {
    ...html.configs["flat/recommended"],
    files: ["**/*.html"],
    rules: {
      "@html-eslint/use-baseline": ["warn", {
        "available": "widely"
      }],
    },
  },
```

**Actual result:**

```sh
58:40  warning  Attribute 'referrerpolicy="origin-when-cross-origin"' on '<a>' is not a widely available baseline feature  @html-eslint/use-baseline
```

<img width="806" alt="Image" src="https://github.com/user-attachments/assets/cbd32d1a-b2e4-4771-8258-b30ff50a200b" />

The warning starts on character position 40, which is the attribute name, `referrerpolicy`.

**Expected result:**

Because the attribute value is unavailable and causing the warning, it should be highlighted, not the attribute name. The character position of the warning should line up with the value, `"origin-when-cross-origin"`.

**Comment by yeonjuan** [comment]
@rviscomi Thank you for the report! I’ll work on fixing it.
